python/packages/autogen-ext/src/autogen_ext/models/openai/_openai_client.py

```diff
@@ -101,12 +103,31 @@
 disallowed_create_args = set(["stream", "messages", "function_call", "functions", "n"])
 required_create_args: Set[str] = set(["model"])
 
+USER_AGENT_HEADER_NAME = "User-Agent"
```

Comment:
We can make `AUTOGEN_USER_AGENT` a module level variable so user can set it to "" to disable usage tracking.

Reply:
Add a note to the API doc under `AzureOpenAIChatCompletionClient` is good

Reply:
True we can do that, but I wouldn't view it as usage tracking in quite that way. For example, the user agent for a web request in your browser is not seen as tracking